frontend/src/pages/stepper.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Stepper from "@material-ui/core/Stepper";
+import Step from "@material-ui/core/Step";
+import StepLabel from "@material-ui/core/StepLabel";
+import StepContent from "@material-ui/core/StepContent";
+import Button from "@material-ui/core/Button";
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+  },
+  button: {
+    marginTop: theme.spacing(1),
+    marginRight: theme.spacing(1),
+  },
+  actionsContainer: {
+    marginBottom: theme.spacing(2),
+  },
+  resetContainer: {
+    padding: theme.spacing(3),
+  },
+}));
+
+function getSteps() {
+  return [
+    <Button style={{ color: "#A80909" }}>Introduction</Button>,
+    <Button style={{ color: "#A80909" }}>Project Assignment</Button>,
+    <Button style={{ color: "#A80909" }}>Response</Button>,
+    <Button style={{ color: "#A80909" }}>Gathered Information</Button>,
+    <Button style={{ color: "#A80909" }}>Stakeholders</Button>,
+    <Button style={{ color: "#A80909" }}>Results</Button>,
+    <Button diabled>Feedback</Button>,
+    <Button diabled>Response</Button>,
+  ];
+}
+
+function getStepContent(step) {
+  switch (step) {
+    case 0:
+      return ``;
+    case 1:
+      return "";
+    case 2:
+      return ``;
+    default:
+      return "";
+  }
+}
+
+export default function VerticalLinearStepper() {
+  const classes = useStyles();
+  const [activeStep, setActiveStep] = React.useState(0);
+  const steps = getSteps();
+
+  return (
+    <div className={classes.root}>
+      <Stepper activeStep={activeStep} orientation="vertical">
+        {steps.map((label, index) => (
```

Comment:
Nice use of .map here, it is something you will want to be using a lot for this project for sure